src/xrpld/app/tx/detail/Transactor.h

```diff
@@ -392,6 +392,15 @@ Transactor::invokePreflight(PreflightContext const& ctx)
     // TODO: If #5650 is merged, use its transaction -> amendment lookup here to
     // do a first-pass check. Rewrite or remove any `isEnabled` overloads that
     // check those default amendments.
+
+    // Using this lookup does NOT require checking the fixDelegateV1_1. The data
+    // exists regardless of whether it is enabled.
+    auto const feature =
+        Permission::getInstance().getTxFeature(ctx.tx.getTxnType());
+
+    if (feature && !ctx.rules.enabled(*feature))
+        return temDISABLED;
+
```

Comment:
With this change (and f85ce9c742) the name `isEnabled` used below becomes misleading; how about renaming it to .... I dunno (naming is hard), perhaps `isEnabledExtra` ?